master/internal/grpc/api.go

```diff
@@ -84,10 +84,6 @@ func RegisterHTTPProxy(e *echo.Echo, port int, enableCORS bool, cert *tls.Certif
 	}
 	handler := func(c echo.Context) error {
 		request := c.Request()
-		if origin := request.Header.Get("Origin"); enableCORS && origin != "" {
-			c.Response().Header().Set("Access-Control-Allow-Origin", origin)
-			c.Response().Header().Set("Access-Control-Allow-Credentials", "true")
-		}
```

Comment:
unneeded because at this point echo middleware has already handled it?

Reply:
correct.

Reply:
and the allow credentials are no longer needed since we stop using cookies essentially.